jina/peapods/pods/k8s.py

```diff
@@ -311,8 +311,12 @@ def port_expose(self) -> int:
 
     @property
     def host(self) -> str:
```

Comment:
I think is better to remove this property, not needed apparently

Reply:
It is needed  when sending data.

Reply:
Gateway host is needed.

Reply:
```
@property
    def client(self) -> 'BaseClient':
        """Return a :class:`BaseClient` object attach to this Flow.

        .. # noqa: DAR201"""

        kwargs = dict(
            host=self.host,
            port=self.port_expose,
            protocol=self.protocol,
        )
        kwargs.update(self._common_kwargs)
        return Client(**kwargs)
```

in base flow